### PR TITLE
Store submission time of SRPM Builds

### PR DESCRIPTION
**Pull request by IceWreck**

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]

**Comment by jpopelka** [comment]
> [jpopelka](/jpopelka) approved these changes

once the tests are green :-)

File: alembic/versions/a0e48c400be4_store_submit_time_of_srpm_builds.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+revision = "a0e48c400be4"
+down_revision = "521433a1fdb0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("celery_tasksetmeta")
+    op.drop_table("celery_taskmeta")
```
Comment (TomasTomecek) [inline alembic/versions/a0e48c400be4_store_submit_time_of_srpm_builds.py:22]: :eyes: 

is this something we really want?

**Review by TomasTomecek — COMMENTED** [review]
+1, thank you!

File: alembic/versions/a0e48c400be4_store_submit_time_of_srpm_builds.py (hunk 1 above)
Comment (IceWreck) [reply alembic/versions/a0e48c400be4_store_submit_time_of_srpm_builds.py:22]: Dont know why this happened. Will remove all of this form the autogenerated script

**Review by csomh — COMMENTED** [review]
I'm not sure if I'm missing something... is `build_submitted_time` set somewhere? :smile:

**Comment by IceWreck** [comment]
> I'm not sure if I'm missing something... is `build_submitted_time` set somewhere? smile

Its value is set when the SRPMBuildModel object is instantiated. `
```
Column(DateTime, default=datetime.utcnow)
```
We do the same thing in CoprBuildModel as well

**Review by TomasTomecek — APPROVED** [review]
well done!

**Comment by csomh** [comment]
> > I'm not sure if I'm missing something... is `build_submitted_time` set somewhere? smile
> 
> Its value is set when the SRPMBuildModel object is instantiated. `

I missed that :facepalm: Thanks!

**Pull request merged**